docs/standards/rest.md

```diff
@@ -201,6 +201,44 @@ The `version` URL query parameter, `?version=version_string` is reserved for sel
 
 `starting_after`, `ending_before`, and `limit` are reserved for cursor pagination on resource collections, as defined in [JSON API Pagination Parameters](../../principles/jsonapi/#pagination-parameters).
 
+### <a id="expansion"></a>Expansion
+
+A resource's relationships may inline the related resource's attributes to conserve API requests and provide a richer response. This inline enrichment of related data attributes is called _expansion_.
+
+#### `expand` query parameter
+
+The `expand` URL query parameter is reserved for expressing expansion of relationships. Resource endpoints are not required to support this parameter, but when they do, they must declare the `expand` parameter schema as an array of enum values. The enum values must match the relationship names provided in the resource data `relationships` map.
+
+The parameter should be declared in OpenAPI with the same representation as other array parameters, `{"style": "form", "explode": false}`.
+
+The default expansion behavior when `expand` is not specified is "no expansion". If the default expansion differs, this behavior must be documented in the parameter's description.
+
+#### Constraints
+
+Resources may support expansion of any one-to-one relationship. They must not expand one-to-many relationships _in relationship data_. The reason for this is complexity around pagination -- collections must be paginated in Snyk APIs, and this becomes complicated with multiple collections within resources, or collections within collections.
+
+It is still possible to represent a one-to-many relationship with expansion, however. A resource may expand relationships in each resource in a collection response.
```

Comment:
This isn't super clear. It'd be helpful for there to be explicit examples of one-to-one and one-to-many and how you'd approach those use cases.

Reply:
I think I need a better canonical concrete example of resources with this kind of relationship. JSON API's `authors` & `articles` would be a lot clearer than `thing` and `this-other-thing` :)

Reply:
Updated